test/java/src/com/ibm/streamsx/topology/test/python/SubscribeSPLDictTest.java

```diff
@@ -99,19 +101,26 @@ public void testSubscribeSPLDictMap() throws Exception {
         	}
 
         	{
-        		Set<?> ex =spl.getSet("si32");
-            	JSONArray pya = (JSONArray) json.get("si32");
-            	assertEquals(ex.size(), pya.size());
-            	
-              /**
-              int j = 0;
-              for (int si : ex) {
-            		assertEquals(si, ((Number) pya.get(j)).intValue());
-                j++;
-              }
-              **/
+        		Set<?> ex = spl.getSet("si32");
+            JSONArray pya = (JSONArray) json.get("si32");
+            assertEquals(ex.size(), pya.size());
+
+            setAssertHelper(ex, pya);
         	}
 
         }
     }
+
+    private <T> void setAssertHelper(Set<T> s, JSONArray pya) {
+      Set<T> sorteds = new TreeSet<T> (s);
```

Comment:
Don't think you need the complexity of a TreeSet. Just create a set from the list and assert that the sets are equal. Something like ...

`assertTrue(ex.equals(new HashSet<Integer>(pya));`